src/Drush/Command/BltDoctorCommand.php

```diff
@@ -199,7 +200,7 @@ protected function setStacksEnabled() {
     }
 
     if (file_exists($this->repoRoot . '/Vagrantfile')
-      && file_exists($this->repoRoot . '/project.local.yml')
+      && file_exists($this->repoRoot . '/blt/project.local.yml')
```

Comment:
I'm going to submit another quick/small PR to update this line if you want to include it.